yarn-project/acir-simulator/src/client/unconstrained_execution.ts

```diff
@@ -54,6 +56,26 @@ export class UnconstrainedFunctionExecution {
       },
       getL1ToL2Message: ([msgKey]) => this.context.getL1ToL2Message(fromACVMField(msgKey)),
       getCommitment: ([commitment]) => this.context.getCommitment(this.contractAddress, commitment),
+      storageRead: async ([slot], [numberOfElements]) => {
```

Comment:
where is the oracle call in noir that tells the VM to call `storageRead`?

Reply:
What do you mean? It is the same as with public execution? The `.read()` ends up calling oracles in here.
https://github.com/AztecProtocol/aztec-packages/blob/master/yarn-project/noir-libs/noir-aztec/src/oracle/storage.nr